### PR TITLE
Messages to inform user of the availability of Tickets

### PR DESCRIPTION
**Pull request by masteradhoc**
My PullRequest comes originally comes from here:
https://theeventscalendar.com/support/forums/topic/no-message-end-date-for-sales-reached/

I think the user should be informed when tickets are no longer or not yet on sale. Also i added a message when there are no Tickets available for this event!. As this only comes up when Event Tickets is installed it should work out for the users of the plugin.

OLD PR:
https://github.com/moderntribe/event-tickets/pull/95
https://github.com/moderntribe/event-tickets/pull/94

Internal ticket reference: https://central.tri.be/issues/45467

### CODE REVIEW

File: src/views/tickets/rsvp.php [hunk 1]
```diff
@@ -143,4 +152,8 @@
 $content = ob_get_clean();
 if ( $is_there_any_product ) {
 	echo $content;
+}else{
```
Comment (borkweb) [inline src/views/tickets/rsvp.php]: @masteradhoc, I wonder if the logic on lines 89-97 would go nicely down here instead? The `Tickets are no longer on sale` message seems like a nice catch-all that would also satisfy the nostock message you have below! Thoughts?

Comment (masteradhoc) [inline src/views/tickets/rsvp.php]: @borkweb I dont mind putting it down, so if its better for you i'll take the lines down today evening :)

**Comment by masteradhoc** [comment]
@borkweb finished my work. :)

**Comment by borkweb** [comment]
We're ready to merge this, @masteradhoc! Thank you for the work on this PR :) I'll take it from here and implement similar messaging in our Event Tickets Plus plugin as well!

**Comment by masteradhoc** [comment]
Thanks @borkweb! Are you able to tell me when this will be released? Also i would be interested in translating this one. Any additional coding needed therefore?

**Comment by borkweb** [comment]
We plan to release this with 4.2 in a couple of months!

**Comment by masteradhoc** [comment]
@borkweb thanks for the info. Any info about the translation question?

**Pull request merged**